data/mods/Xedra_Evolved/mapgen/safehouse.json

```diff
@@ -4,7 +4,6 @@
     "om_terrain": "safe_house",
     "//": "used by safe_house_basement",
     "type": "mapgen",
-    "weight": 10,
```

Comment:
Ok so if the weight here is not useful how do I make sure that this is a very rare house spawn? I understand this likely means this has been spawning more than I'd like already.

Reply:
The weight of 5 in https://github.com/CleverRaven/Cataclysm-DDA/blob/394f235c205b8b72f39d86ee79b66a2e0f92c6d3/data/mods/Xedra_Evolved/regional_overlay.json#L5
is what defines how often it spawns which comparing it to
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/regional_map_settings.json#L834-L1006
makes it pretty rare